### PR TITLE
ENGG-2263: Show Captured localStorage events in session details panel

### PR DESCRIPTION
**Pull request by RuntimeTerror10**

### CODE REVIEW

File: app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  return (
+    <div className="session-storage-logs-container">
+      <Table
+        className="session-storage-logs-table"
+        dataSource={events.storage || []}
+        columns={columns}
+        pagination={false}
+        // TEMP
+        scroll={{ y: "calc(100vh - 210px)" }}
```
Comment (nafees87n) [inline app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx:77]: Will this be removed?

File: app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      {
+        title: `Old value`,
+        key: "oldValue",
+        width: 140,
+        render: (_: any, record: StorageEventData) => (
+          <div className="session-storage-log-cell">
+            {record.eventType === "initialStorageValue" ? "-" : record.oldValue || "-"}
+          </div>
+        ),
+      },
```
Comment (nafees87n) [inline app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx:65]: Table should handle empty strings, there should be a placeholder value
image

File: app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx (hunk 1 above)
Comment (RuntimeTerror10) [reply app/src/features/sessionBook/components/SessionViewerBottomSheet/components/SessionStorageLogs/SessionStorageLogs.tsx]: This is fixed now

File: app/src/features/sessionBook/screens/DraftSessionScreen/components/SessionTrimmer/SessionTrimmer.tsx
[the remark this answers is not in the document]
Comment (RuntimeTerror10) [reply app/src/features/sessionBook/screens/DraftSessionScreen/components/SessionTrimmer/SessionTrimmer.tsx:1]: We are not trimming the storage sessions for now

**Review by nafees87n — APPROVED** [review]

File: app/package.json [hunk 3]
```diff
@@ -56,6 +56,7 @@
     "@reduxjs/toolkit": "^1.9.0",
     "@requestly/mock-server": "0.1.6",
     "@requestly/requestly-core": "file:..",
+    "@requestly/web-sdk": "^0.15.1",
```
Comment (nafees87n) [inline app/package.json:59]: Why move it to dependencies? We only consume types from here

**Pull request merged**